hs_discover/views.py

```diff
@@ -192,20 +205,27 @@ def get(self, request, *args, **kwargs):
             if result.creator:
                 creator = result.creator
 
+            authors = creator  # there is no concept of authors in DataOne standard
+            # authors might be string 'None' here
+
             if result.author:
                 author_link = result.author_url
                 author = str(result.author)
+                if authors == 'None':
+                    authors = author  # author would override creator in
             else:
-                if creator != 'None':
-                    try:
-                        if creator[0]:
-                            author = creator[0]
-                        elif creator[1]:
-                            author = creator[1]
-                    except Exception as gen_creator_ex:
-                        pass  # look for nonempty first or second creator based on corrupted data in production
-                elif result.organization:
-                    author = result.organization
+                if result.organization:
+                    if isinstance(result.organization, list):
+                        author = str(result.organization[0])
+                    else:
+                        author = str(result.organization)
+
+                    author = author.replace('"', '')
+                    author = author.replace('[', '')
+                    author = author.replace(']', '').strip()
+
+                    if authors == 'None':
+                        authors = author
 
```

Comment:
handle dirty data that contains strings of lists

Reply:
I'm just gonna have to trust you on a lot of this dirty data handling.  I do know the data is dirty and it is difficult to deal with.

Reply:
Our stakeholders will let us know